CrossAppLoginUI/src/main/kotlin/com/infomaniak/core/crossloginui/views/CrossLoginListAccountsView.kt

```diff
@@ -0,0 +1,117 @@
+
+    private var onAccountClickedListener: ((account: CrossLoginUiAccount) -> Unit)? = null
+    private var onAnotherAccountClickedListener: (() -> Unit)? = null
+    private var onCloseClicked: (() -> Unit)? = null
+
+    private var primaryColor by mutableStateOf<Color?>(null)
+    private var onPrimaryColor by mutableStateOf<Color?>(null) // TODO
+    private var titleColor: Color? = null
+    private var descriptionColor: Color? = null
+    private var avatarStrokeColor: Color? = null
+    private var buttonStrokeColor: Color? = null
+
+    init {
+        context.obtainStyledAttributes(attrs, R.styleable.CrossLoginListAccountsView, defStyleAttr, 0).apply {
+            primaryColor = getColorOrNull(R.styleable.CrossLoginListAccountsView_crossLoginPrimaryColor)
+            onPrimaryColor = getColorOrNull(R.styleable.CrossLoginListAccountsView_crossLoginOnPrimaryColor)
+            titleColor = getColorOrNull(R.styleable.CrossLoginListAccountsView_crossLoginTitleColor)
+            descriptionColor = getColorOrNull(R.styleable.CrossLoginListAccountsView_crossLoginDescriptionColor)
+            avatarStrokeColor = getColorOrNull(R.styleable.CrossLoginListAccountsView_crossLoginAvatarStrokeColor)
+            buttonStrokeColor = getColorOrNull(R.styleable.CrossLoginListAccountsView_crossLoginButtonStrokeColor)
+            recycle()
+        }
+    }
+
+    @Composable
+    override fun Content() {
+        MaterialThemeFromXml {
+
+            val colors = CrossLoginDefaults.colors(
+                primary = primaryColor,
+                onPrimary = onPrimaryColor,
+                title = titleColor,
+                description = descriptionColor,
+                avatarStroke = avatarStrokeColor,
+                buttonStroke = buttonStrokeColor,
+            )
+
+            CrossLoginListAccounts(
+                accounts = { accounts },
+                colors = colors,
+                onAccountClicked = { onAccountClickedListener?.invoke(it) },
+                onAnotherAccountClicked = { onAnotherAccountClickedListener?.invoke() },
+                onCloseClicked = { onCloseClicked?.invoke() },
+            )
+        }
+    }
+
+    fun setPrimaryColor(@ColorInt newPrimaryColor: Int) {
+        primaryColor = Color(newPrimaryColor)
+    }
+
+    fun setOnPrimaryColor(@ColorInt newOnPrimaryColor: Int) {
+        onPrimaryColor = Color(newOnPrimaryColor)
+    }
+
+    fun setAccounts(list: List<CrossLoginUiAccount>) {
+        accounts.apply {
+            clear()
+            addAll(list)
+        }
+    }
+
+    fun setOnAccountClickedListener(listener: (CrossLoginUiAccount) -> Unit) {
+        onAccountClickedListener = listener
+    }
+
+    fun setOnAnotherAccountClickedListener(listener: () -> Unit) {
+        onAnotherAccountClickedListener = listener
+    }
+
+    fun setOnCloseClickedListener(listener: () -> Unit) {
+        onCloseClicked = listener
+    }
```

Comment:
Should it not be "onSave" instead?

Reply:
I asked iOS to make sure but yeah It's supposed to save only when the save button is hit and you could swipe the bottom sheet down to undo your modifications. Otherwise the save button would make no sense